internal/suites/Standalone/configuration.yml

```diff
@@ -3,6 +3,8 @@
 ###############################################################
 
 port: 9091
+ssl_cert: /var/lib/authelia/ssl/cert.pem
```

Comment:
After second thought, I'll replace `ssl_cert` and `ssl_key` by `tls_cert` and `tls_key`.